src/renderers/dom/shared/__tests__/ReactServerRendering-test.js

```diff
@@ -549,4 +661,27 @@ describe('ReactDOMServer', () => {
       );
     }).toThrowError(/Cannot assign to read only property.*/);
   });
+
+  describe('basic rendering', function() {
+    itRenders('should render a blank div', render =>
+      render(<div />).then(e => expect(e.tagName.toLowerCase()).toBe('div')));
+
+    itRenders('should render a div with inline styles', render =>
+      render(<div style={{color:'red', width:'30px'}} />).then(e => {
+        expect(e.style.color).toBe('red');
+        expect(e.style.width).toBe('30px');
+      })
+    );
+
+    itRenders('should render a self-closing tag', render =>
+      render(<br />).then(e => expect(e.tagName.toLowerCase()).toBe('br')));
```

Comment:
(I think tagName is guaranteed to be uppercase so this could just be expect(e.tagName).toBe('BR')?)